graph/typeDefs.graphql

```diff
@@ -290,6 +293,28 @@ type FlagAction implements Action {
   created_at: Date
 }
 
+# A DONTAGREE action that contains do not agree metadata.
+type DontAgreeAction implements Action {
+
+  # The ID of the DontAgree Action.
+  id: ID!
+
+  # The reason for which the DontAgree Action was created.
+  reason: String
```

Comment:
In principal we might have multiple "don't agree" reasons in the future, and in that case I wouldn't want to put a schema change. If that's not a concern then I'm happy to kill it.